ofctrl/ofAction.go

```diff
@@ -282,9 +284,9 @@ type SetDSCPAction struct {
 	Value uint8
 }
 
-func (a *SetDSCPAction) GetActionMessage() openflow13.Action {
-	field := openflow13.NewIpDscpField(a.Value)
-	return openflow13.NewActionSetField(*field)
+func (a *SetDSCPAction) GetActionMessage() openflow15.Action {
```

Comment:
Since load action is not supported from OF 1.5, these setXXAction should enhance to support setting mask value which is used to present the range used in load action ? At least, SetDSCPAction is required to support setting mask.